pkgs/development/libraries/opencv/4.x.nix

```diff
@@ -274,15 +335,15 @@ stdenv.mkDerivation {
       "$out/lib/pkgconfig/opencv4.pc"
   '';
 
-  hardeningDisable = [ "bindnow" "relro" ];
+  # hardeningDisable = [ "bindnow" "relro" ];
```

Comment:
Whats with this?

Reply:
I just commented it, I'll remove this line.